frontend/src/constant/getObserverTemplate.ts

```diff
@@ -0,0 +1,7 @@
+export function getObserverTemplate(done: any) {
+  return {
+    next: (game: any) => {},
```

Comment:
the heck is dis

Reply:
nezinau kaip kitaip padaryt observer sablona, kad galetu implementuot tik next funkcija observery